examples/cloud-router-2-aws-connection/main.tf

```diff
@@ -0,0 +1,41 @@
+terraform {
+  required_providers {
+    equinix = {
+      source  = "equinix/equinix"
+      version = ""
+    }
+  }
+}
+
+provider "equinix" {
+  client_id     = var.equinix_client_id
+  client_secret = var.equinix_client_secret
+}
+
+module "cloud_router_port_connection" {
+  source = "../../modules/cloud-router-connection"
+
+  connection_name       = var.connection_name
+  connection_type       = var.connection_type
+  notifications_type    = var.notifications_type
+  notifications_emails  = var.notifications_emails
+  additional_info       = [{ key = "accessKey", value = var.access_key }, { key = "secretKey", value = var.secret_key }]
+  bandwidth             = var.bandwidth
+  redundancy            = var.redundancy
+  purchase_order_number = var.purchase_order_number
+
+  #Aside
+  fcr_uuid      = var.fcr_uuid
```

Comment:
Aside variables should be contextual as well.

Use the following replacements:
`aside_fcr_uuid`